src/formatDate.js

```diff
@@ -8,7 +8,38 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
+  const toSeparator = toFormat.pop();
```

Comment:
Similarly to the previous comment, Array.prototype.pop() is used here, which mutates the `toFormat` array. It's better to avoid mutating input parameters.

Reply:
The same issue here